src/diagnostic-channel-publishers/package.json

```diff
@@ -4,16 +4,15 @@
   "main": ".dist/src/index.js",
   "types": ".dist/src/index.d.ts",
   "scripts": {
-    "build": "tsc --outDir ./.dist && node copyTestAssets.js",
-    "lint": "tslint -c tslint.json -p tsconfig.json",
+    "build": "node node_modules/typescript/bin/tsc --outDir ./.dist && node copyTestAssets.js",
```

Comment:
That's strange, since that exact scenario (no global tslint) works for me:

```
diagnostic-channel>tslint
'tslint' is not recognized as an internal or external command,
operable program or batch file.

diagnostic-channel>npm run lint

> diagnostic-channel@0.1.0 lint \diagnostic-channel
> tslint -c tslint.json -p tsconfig.json

\diagnostic-channel>npm -v
4.1.2
```
But if it was needed for travis, then that's fine with me.